src/script/script.cpp

```diff
@@ -204,6 +204,15 @@ unsigned int CScript::GetSigOpCount(const CScript& scriptSig) const
     return subscript.GetSigOpCount(true);
 }
 
+bool CScript::IsPayToAnchor() const
+{
+    return (this->size() == 4 &&
+        (*this)[0] == OP_TRUE &&
+        (*this)[1] == 0x02 &&
```

Comment:
whats the `0x02`?
P2Anchor is `OP_TRUE <0x4e73>`

Reply:
It's the instruction to push next two bytes on the stack